src/test/java/uk/gov/hmcts/divorce/print/BulkPrintServiceTest.java

```diff
@@ -110,5 +186,154 @@ void shouldReturnLetterIdForValidRequest() {
                 entry(CASE_REFERENCE_NUMBER_KEY, "5678"),
                 entry(CASE_IDENTIFIER_KEY, "1234")
             );
+
+        verify(httpServletRequest, times(2))
+            .getHeader(AUTHORIZATION);
+        verify(idamService, times(2))
+            .retrieveUser(APP_1_SOL_AUTH_TOKEN);
+        verify(documentManagementClient).downloadBinary(
+            APP_1_SOL_AUTH_TOKEN,
+            TEST_SERVICE_AUTH_TOKEN,
+            solicitorRolesCsv,
+            userId,
+            documentUuid
+        );
+        verify(authTokenGenerator).generate();
+
+    }
+
+    private User solicitorUser(List<String> solicitorRoles, String userId) {
+        UserDetails userDetails = UserDetails
+            .builder()
+            .roles(solicitorRoles)
+            .id(userId)
+            .build();
+
+        return new User(APP_1_SOL_AUTH_TOKEN, userDetails);
+    }
```

Comment:
private method goes to last after all public methods

Reply:
moved private method to the bottom